content/zh/docs/concepts/architecture/garbage-collection.md

```diff
@@ -0,0 +1,181 @@
+
+### 前景级联删除 {#foreground-deletion}
+
+在前台级联删除中，要删除的所有者对象首先进入
+a*正在删除*状态。在此状态下，以下情况发生在
+所有者对象：
+
+	* Kubernetes API服务器设置对象的` metadata.deletionTimestamp'`
+字段设置为对象标记为删除的时间。
+	* Kubernetes API服务器还将“metadata.finalizers”字段设置为
+`前景删除`。
+	* 在删除之前，对象通过Kubernetes API保持可见
+过程已经完成。
+
+所有者对象进入删除进行中状态后，控制器
+删除依赖项。删除所有从属对象后，控制器
+删除所有者对象。此时，该对象在视图中不再可见
+Kubernetes API。
+
+在前台级联删除期间，块所有者的唯一依赖项
+删除是具有“ownerReference.BlockOwnerDelete=true”字段的删除。
+请参阅[使用前台级联删除]（/docs/tasks/administrate cluster/Use cascading deletation/#Use front cascading deletation）
+了解更多。
+
+### 后景级联删除 {#background-deletion}
+
+在后台级联删除中，Kubernetes API服务器删除所有者
+对象，并且控制器将清除中的从属对象
+背景。默认情况下，Kubernetes使用后台级联删除，除非
+手动使用前景删除或选择孤立从属对象。
+
+请参阅[使用后台级联删除]（/docs/tasks/administrate cluster/Use cascading deletion/#Use background cascading deletion）
+了解更多。
+### 独立拥有者
+
+当Kubernetes删除所有者对象时，将调用留下的依赖项
+*孤立*对象。默认情况下，Kubernetes删除依赖对象。学习如何
+要覆盖此行为，请参阅[删除所有者对象和孤立从属对象]（/docs/tasks/administrate cluster/use cascading deletation/#set orphan deletation policy）。
+
+## 未使用容器和镜像的垃圾收集 {#containers-images}
+
+{{<glossary\u tooltip text=“kubelet”term\u id=“kubelet”>}执行垃圾处理
+每五分钟收集一次未使用的图像，每五分钟收集一次未使用的容器
+分钟您应该避免使用外部垃圾收集工具，因为它们可以
+打破kubelet行为并移除应该存在的容器。
+
+要为未使用的容器和图像垃圾收集配置选项，请调整
+kubelet使用[配置文件]（/docs/tasks/administrate cluster/kubelet config file/）
+并使用
+[`KubeletConfiguration`]（/docs/reference/config-api/kubeletconfig.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration）
+资源类型。
+
+### 容器映像生命周期
+
+Kubernetes通过其*镜像管理器*管理所有图像的生命周期，
+这是kubelet的一部分，与cadvisor合作。库贝莱酒店
+在进行垃圾收集时考虑以下磁盘使用限制
+决定：
+
+  * `高阈值百分比`
+  * `低阈值百分比`
+
+磁盘使用率高于配置的“HighThresholdPercent”值会触发垃圾
+集合，它根据上次使用图像的时间顺序删除图像，
+先从最老的开始。kubelet删除图像
+直到磁盘使用率达到“LowThresholdPercent”值。
+
+### 容器映像垃圾回收 {#container-image-garbage-collection}
+
+kubelet垃圾根据以下变量收集未使用的容器，
+您可以定义：
+
+  * `MinAge`: kubelet可以收集垃圾的最低年龄
+容器通过设置为“0”禁用。
+  * `MaxPerPodContainer`: 每个吊舱对的最大死容器数
+可以有。通过设置为小于“0”来禁用。 
+  * `MaxContainers`: 群集可以拥有的最大死容器数。
+通过设置为小于“0”来禁用。 
+
+除了这些变量外，kubelet垃圾收集未识别的
+已删除的容器，通常从最旧的容器开始。
+
+`MaxPerPodContainer`和MaxContainer`可能会相互冲突
+在保持每个吊舱最大容器数量的情况下
+（`MaxPerPodContainer`）将超出允许的全局死亡总数
+容器（`MaxContainers`）。在这种情况下，kubelet会进行调整
+`MaxPodPerContainer`解决冲突。最坏的情况是
+将“MaxPerPodContainer”降级为“1”，并逐出最旧的容器。
+此外，已删除的POD所拥有的容器将被删除一次
+它们比“米纳奇”古老。
+
+{{<note>}}
+kubelet only垃圾收集它管理的容器。
+{{</note>}}
+
+## 垃圾是收集配置 {#configuring-gc}
+
+您可以通过配置特定于的选项来优化资源的垃圾回收
+管理这些资源的控制器。以下几页向您展示了如何
+配置垃圾收集：
+
+  * [配置Kubernetes对象的级联删除](/docs/tasks/administer-cluster/use-cascading-deletion/)
+  * [配置已完成作业的清理](/docs/concepts/workloads/controllers/ttlafterfinished/)
+  
+<!-- * [配置未使用的容器和映像垃圾回收 ](/docs/tasks/administer-cluster/reconfigure-kubelet/) -->
+
+## {{% heading "whatsnext" %}}
+
+* 了解更多关于 [ownership of Kubernetes objects](/docs/concepts/overview/working-with-objects/owners-dependents/).
+* 了解更多关于 Kubernetes [finalizers](/docs/concepts/overview/working-with-objects/finalizers/).
+* 了解更多关于 [TTL controller](/docs/concepts/workloads/controllers/ttlafterfinished/) (beta) that cleans up finished Jobs.
```

Comment:
I'd link to localized pages, eg, https://kubernetes.io/zh/docs/concepts/overview/working-with-objects/owners-dependents/

Also “that cleans up finished Jobs” should be localized, even if there's no Chinese version of https://k8s.io/docs/concepts/workloads/controllers/ttlafterfinished/ available

Reply:
> 

I have modified it